packages/patternfly-3/react-console/src/VncConsole/VncConsole.js

```diff
@@ -62,17 +63,17 @@ class VncConsole extends React.Component {
   }
 
   componentWillUnmount() {
-    this.removeEventListeners();
     this.disconnect();
+    this.removeEventListeners();
+    this.rfb = undefined;
```

Comment:
With this change, the `RFB.disconnect()` is skipped. Can you please explain reason for that?

Reply:
But no, it doesn't skipped. Line 66 perform disconnecting. And previously `onDisconnected` event doesn't work, because it removed all listeners before it disconnected.